.readthedocs.yml

```diff
@@ -1,15 +1,11 @@
-# .readthedocs.yml
-# Read the Docs configuration file
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
-# Required
 version: 2
 
-# Build documentation in the docs/ directory with Sphinx
 sphinx:
   configuration: doc/conf.py
 
+# Optionally set the version of Python and requirements required to build your docs
 python:
   version: 3.7
   install:
-    - requirements: doc/requirements.txt
+    - requirements: doc/.sphinx/requirements.txt
```

Comment:
I don't especially mind, but what was wrong with the old location? It's the requirements for building the Python project rooted in `doc`, it seems reasonable to have them there.

Reply:
Similarly for the templates etc. I guess you're trying to hide the sphinx stuff from people? Not worth it IMO but I don't mind being consistent.

Reply:
Yeah I was trying to hide as much as possible. I tried to hide conf.py too, but that didn't see possible, something to do with the way the python config references things in a path. It won't allow `../something`.

Reply:
Don't particularly care about consistency of location of requirements, happy to leave that to the owner of the repo. So long as everything except README.rst is under /doc

Reply:
FWIW I don't feel the need to hide that in our case, I'd actually rather have it be more visible. So I think I'm going to put it back if you don't mind. Shouldn't affect anything else.